src/test/ui/feature-gates/feature-gate-relaxed_struct_unsize.rs

```diff
@@ -1,10 +0,0 @@
-// Test that we allow unsizing even if there is an unchanged param in the
-// field getting unsized.
-struct A<T, U: ?Sized + 'static>(T, B<T, U>);
-struct B<T, U: ?Sized>(T, U);
-
-fn main() {
-    let x: A<[u32; 1], [u32; 1]> = A([0; 1], B([0; 1], [0; 1]));
-    let y: &A<[u32; 1], [u32]> = &x; //~ ERROR mismatched types
```

Comment:
I assume this was deleted because the error no longer occurs? Should this be turned into a check-pass test then?

Reply:
we already have this as a check-pass test, this was only to check that the feature gate is actually needed

Reply:
Ah, excellent. Thanks!